components/ucd/normal/tests/unicode_version_tests.rs

```diff
@@ -20,3 +21,149 @@ fn test_version_against_ucd_core() {
         unic_ucd_core::UNICODE_VERSION
     );
 }
+
```

Comment:
`tests/unicode_version_tests.rs` files are only to do checks on Unicode Version matters.

We need to move this test to a new file, like `tests/conformance_tests.rs`, similar to [`ucd-bidi` conformance tests](https://github.com/behnam/rust-unic/blob/master/components/bidi/tests/conformance_tests.rs).